packages/discord.js/typings/index.d.ts

```diff
@@ -677,7 +677,7 @@ export class BaseGuildVoiceChannel extends GuildChannel {
   public bitrate: number;
   public get full(): boolean;
   public get joinable(): boolean;
-  public get members(): Collection<Snowflake, GuildMember>;
+  public get members(): Collection<Snowflake, GuildMember | PartialGuildMember>;
```

Comment:
Since this is filtered from cache it won't contain `PartialGuildMember` anymore.